src/test/java/guiunittests/PersonEditDialogUnitTest.java

```diff
@@ -0,0 +1,86 @@
+package guiunittests;
+
+import address.TestApp;
+import address.controller.PersonEditDialogController;
+import address.testutil.TypicalTestData;
+
+import guitests.GuiRobot;
+import guitests.guihandles.EditPersonDialogHandle;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * To test the basic features of the edit dialog
+ */
+public class PersonEditDialogUnitTest extends GuiUnitTestBase {
```

Comment:
Is this the only guiUnitTest we have? If not, others need to extend from the same parent class.